### ISSUE TITLE
Error while trying to get Payment state

### ISSUE THREAD

**Issue by joarobles** [pagos360/sdk-php-v0#2]
When I receive a notification through webhooks I try to get the payment state in order to update my records like this:

```
$pagos360 = new Pagos360\Pagos360($apiKey, $endpoint);
$paymentRequest = $pagos360->retrievePaymentRequest($paymentId);
$state = $paymentRequest->getState();
```

But I get the following error:
`PHP Fatal error:  Call to a member function getName() on a non-object in /vendor/pagos360/sdk-php/src/Model/PaymentRequest.php on line 359`

Any idea?

**Comment by Pnoexz** [comment]
@joarobles This looks like an issue related to #1. Can you try `$state = $paymentRequest->state;`?

**Comment by ghost** [comment]
This issue has been fixed in version [0.1.2](https://github.com/pagos360/sdk-php/releases/tag/0.1.2)
